ui-core/src/styles/inputs/slider.css

```diff
@@ -0,0 +1,72 @@
+.range-wrapper {
+  margin: 50px;
+  width: 80%;
+  height: 10px;
+
+  .range-slider {
+    -webkit-appearance: none;
+    appearance: none;
+    height: 10px;
+    width: 100%;
+  
+    &::-webkit-slider-runnable-track {
+      -webkit-appearance: none;
+      appearance: none;
+      height: 4px;
+      border-radius: 2px;
+      box-shadow: 0px 0px 0px 0.5px rgba(0, 0, 0, 0.25) inset;
+      background-color: rgba(0, 0, 0, 0.05);
+    }
+    
+    &::-moz-range-track {
```

Comment:
Since its the exact same styles as webkit, can't we do (same for the thumb)
```
&::-webkit-slider-runnable-track, ::-moz-range-track {
...
}
```

Reply:
couldn’t get it to work